platform/windows/os_windows.cpp

```diff
@@ -805,7 +805,12 @@ void OS_Windows::process_key_events() {
 
 				k->set_pressed(ke.uMsg == WM_KEYDOWN);
 
-				k->set_scancode(KeyMappingWindows::get_keysym(ke.wParam));
+				if ((ke.lParam & (1 << 24)) && (ke.wParam == VK_RETURN)) {
```

Comment:
I wonder, might it be a performance concern to check for this on every single key event? Isn't there a better way to properly get KEY_ENTER recognized by `KeyMappingWindows::get_keysym`?